src/js/components/awardv2/table/FederalAccountTable.jsx

```diff
@@ -78,6 +78,9 @@ export default class FedAccountTable extends React.Component {
         if (column === 'fedAccount') {
             isLink = item.accountNumber && `#/federal_account/${item.accountNumber}`;
         }
+        if (column === 'awardingAgencyName') {
+            isLink = item.awardingAgencyName && `#/agency/${item.awardingAgencyId}`;
```

Comment:
The first conditional should check for `item.awardingAgencyId`, so that if the id is falsy `isLink` will resolve to false.

Reply:
That is a good check, are there agencies without id's?